runtime/manager.go

```diff
@@ -0,0 +1,174 @@
+package runtime
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/benbjohnson/clock"
+	"github.com/mitchellh/mapstructure"
+	typesCons "github.com/pokt-network/pocket/consensus/types"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/spf13/viper"
+)
+
+var _ modules.RuntimeMgr = &Manager{}
+
+type Manager struct {
+	config  *runtimeConfig
+	genesis *runtimeGenesis
+
+	clock clock.Clock
+}
+
+func NewManagerFromFiles(configPath, genesisPath string, options ...func(*Manager)) *Manager {
+	mgr := &Manager{
+		clock: clock.New(),
+	}
+
+	cfg, genesis, err := mgr.init(configPath, genesisPath)
+	if err != nil {
+		log.Fatalf("[ERROR] Failed to initialize runtime builder: %v", err)
+	}
+	mgr.config = cfg
+	mgr.genesis = genesis
+
+	for _, o := range options {
+		o(mgr)
+	}
+
+	return mgr
+}
+
+func NewManagerFromReaders(configReader, genesisReader io.Reader, options ...func(*Manager)) *Manager {
```

Comment:
We could potentially remove this. I have used during development and forgot to clean after myself